### PR TITLE
Re-add admin audit filter

### PR DESCRIPTION
**Pull request by herojan**
It seems to have been removed at some point

### CODE REVIEW

**Comment by cjbooms** [comment]
👍

File: src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala [hunk 1]
```diff
@@ -99,12 +99,11 @@ class ApiValidationSpec
   it should "ensure that all admin routes have extra auditing enabled" in {
     synchRequest(ValidSynchRequest.payload) ~> Route.seal(createRoutesFromDerivations(ingressDerivationLogic)) ~> check {
       val ingressii = responseAs[TestSynchResponse].ingressii
-      ingressii
-        .filter(_.name.contains("jblogs"))
-        .map(_.filters.get)
-        .foreach { filterChain =>
-          filterChain should include("""enableAccessLog() -> unverifiedAuditLog("https://identity.zalando.com/managed-id")""")
```
Comment (irishshagua) [inline src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala]: How was this passing previously? There was nothing in the filter?

Comment (herojan) [reply src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala]: Yeah the list was empty

**Comment by cjbooms** [comment]
👍

File: src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala (hunk 1 above)
Comment (irishshagua) [reply src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala]: 😭 
Shoddy testing....

**Comment by herojan** [comment]
👍

File: src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala (hunk 1 above)
Comment (herojan) [reply src/it/scala/ie/zalando/fabric/gateway/ApiValidationSpec.scala]: Cowboys Ted!

**Pull request merged**